src/templates/fr/2020/index.html

```diff
@@ -6,11 +6,19 @@
 {% block twitter_image_alt %}Web Almanac {{ year }}{% endblock %}
 
 {% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
-{% block date_modified %}2020-06-27T00:00:00.000Z{% endblock %}
+{% block date_modified %}2020-07-03T00:00:00.000Z{% endblock %}
 
-{% block intro_title %}Web Almanac{% endblock %}
-{% block intro_sub_title %}Rapport annuel<br>de HTTP Archive sur<br><b>l’état du Web</b>{% endblock %}
+{% block intro_title %}Coming soon!{% endblock %}
+{% block intro_sub_title %}The 2020 <b>state of the web</b> report{% endblock %}
 
 {% block mission %}
-Bientôt disponible...
+<p>
+  Experts from around the web community are currently hard at work planning, analyzing, and writing content for the 2020 edition of the Web Almanac, which is on track to be published at the end of the year.
+</p>
+
+<p>
+  There are many opportunities available for new contributors to join the project, including: authors, reviewers, analysts, editors, translators, and developers. If you're interested in contributing to the 2020 edition, we'd love to have you!
+</p>
+
+<a href="https://github.com/HTTPArchive/almanac.httparchive.org#contributing" class="btn">Join the team</a>
```

Comment:
Same here - change `Join the team!` to `Rejoignez les équipe! ` since that was translated already in `contributors.html`